model/alertrecord/alert_bookkeeping.go

```diff
@@ -183,6 +184,16 @@ func FindBySpawnHostExpirationWithHours(hostID string, hours int) (*AlertRecord,
 	return FindOne(db.Query(q).Limit(1))
 }
 
+// FindByTemporaryExemptionExpirationWithHours finds the most recent alert
+// record for a spawn host's temporary exemption that is about to expire.
+func FindMostRecentByTemporaryExemptionExpirationWithHours(hostID string, hours int) (*AlertRecord, error) {
```

Comment:
I might suggest "FindByMostRecentTemporary..." (also comment name doesn't match right now)

Reply:
Done, and fixed!